code/modules/mob/living/carbon/carbon.dm

```diff
@@ -259,8 +259,19 @@
 
 			// BEGIN HUGCODE - N3X
 			else
-				playsound(get_turf(src), 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
 				if(M.zone_selected == "head")
+					if(isvox(src)) // Vox quills are sharp
+						M.visible_message(\
+						"<span class='danger'>[M] stabs [p_them()]self trying to pat [src] on the head!</span>",\
+						"<span class='userdanger'>You stab yourself trying to pat [src] on the head!</span>",\
+						)
+						playsound(get_turf(src), 'sound/weapons/bladeslice.ogg', 50, 1, -1)
+						var/mob/living/carbon/human/H = M
+						var/obj/item/organ/external/active_hand = H.get_organ("[H.hand ? "l" : "r"]_hand")
+						active_hand.receive_damage(5, 0)
+						H.update_icons()
+						return // Skip the normal patting code
+					playsound(get_turf(src), 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
```

Comment:
This means regular hugs wont make the thudswoosh sound effect anymore, please fix.

Or alternatively, dont make shitty meme PRs.